CONTRIBUTING.md

```diff
@@ -115,15 +123,26 @@ builds:
 
 ### Write code
 
-TODO
+Here are some suggestions for you while implementing your feature...
```

Comment:
Type class is well documented in the typeclass.md, I'd say for now just remove the content of this section and provide a link to the https://typelevel.org/cats/guidelines.html
e.g. 
```markdown
[See guidelines](/cats/guidelines.html)
```